storey/flow.py

```diff
@@ -629,3 +664,150 @@ def build_flow(steps):
+        for name, bucket in aggregation_element.aggregation_buckets.items():
+            # Only save raw aggregates, not virtual
+            if bucket.should_persist:
+                blob = pickle.dumps(bucket.to_dict())
+                data[name] = {'B': base64.b64encode(blob).decode('ascii')}
+
+        return data
+
+    async def load_store(self):
+        pass
+
+    # Loads a specific key from the store, and returns it in the following format
+    # {
+    #   'feature_name_1': {'first_bucket_time': <time> 'values': []},
+    #   'feature_name_2': {'first_bucket_time': <time> 'values': []}
+    # }
+    async def load_key(self, key):
+        connector = aiohttp.TCPConnector()
+        client_session = aiohttp.ClientSession(connector=connector)
+
+        request_body = json.dumps({'AttributesToGet': '*'})
+
+        try:
+            response = await client_session.put(f'{self._webapi_url}/{self.table}/{key}',
+                                                headers=self._get_item_headers, data=request_body, ssl=False)
+            body = await response.text()
+            if response.status == 404:
+                return None
+            elif response.status == 200:
+                parsed_response = _v3io_parse_get_item_response(body)
+                for name, blob in parsed_response.items():
+                    parsed_response[name] = pickle.loads(blob)
+
+                return parsed_response
+            else:
+                raise V3ioError(f'Failed to get item. Response status code was {response.status}: {body}')
+        except BaseException as ex:
+            raise ex
+        finally:
+            await client_session.close()
+
+    # Deletes the entire table
+    async def delete(self):
+        connector = aiohttp.TCPConnector()
+        client_session = aiohttp.ClientSession(connector=connector)
+        has_more = True
+        next_marker = ''
+        try:
+            while has_more:
+                get_items_body = {'AttributesToGet': '__name', 'Marker': next_marker}
+                response = await client_session.put(f'{self._webapi_url}/{self.table}/',
+                                                    headers=self._get_items_headers, data=json.dumps(get_items_body), ssl=False)
+                body = await response.text()
+                if response.status == 200:
+                    res = _v3io_parse_get_items_response(body)
+                    for item in res['Items']:
+                        await self._delete_item(f'{self._webapi_url}/{self.table}/{item["__name"]}', client_session)
+
+                    has_more = 'NextMarker' in res
+                    if has_more:
+                        next_marker = res['NextMarker']
+                elif response.status != 404:
+                    raise V3ioError(f'Failed to delete table {self.table}. Response status code was {response.status}: {body}')
+
+            await self._delete_item(f'{self._webapi_url}/{self.table}/', client_session)
```

Comment:
We may want to do something smarter here. If the table is concurrently being written to, this will fail, confusingly for the user.